OracleWebLogic/README.md

```diff
@@ -125,4 +81,4 @@ All scripts and files hosted in this project and GitHub [`docker/OracleWebLogic`
 We support WebLogic Server in certified Docker containers, please read our Support statement. For additional details on the most current WebLogic Server supported configurations, please refer to the [Oracle Fusion Middleware Certification Pages](http://www.oracle.com/technetwork/middleware/ias/oracleas-supported-virtualization-089265.html).
 
 ## Copyright
-Copyright (c) 2014-2018 Oracle and/or its affiliates. All rights reserved.
+Copyright (c) 2014-2025 Oracle and/or its affiliates. All rights reserved.
```

Comment:
We should update the copyright to match the current style, i.e., "Copyright (c) 2014, 2025, Oracle and/or its affiliates."